src/lib/ui/AccessCodeForm/AccessCodeForm.tsx

```diff
@@ -173,7 +173,7 @@ function Content({
       : null
 
   const hasCodeInputs =
-    accessCode?.type !== 'time_bound' || accessCode.is_offline_access_code
+    accessCode?.type !== 'time_bound' && accessCode?.is_offline_access_code
```

Comment:
Won't this new condition mean that the UI will only allow editing offline access codes? What about editing online access codes?

Reply:
https://hello-seam.slack.com/archives/C04FS583FJA/p1724680131426929

Reply:
@mastafit I am looking at the thread, but it doesn't answer my original question. Can you answer my original question per the intent of this change?

Reply:
@razor-x You are right, I think we can add extra conditions

Reply:
I mean remove the extra conditions, something like that:
`const hasCodeInputs = accessCode?.type !== 'time_bound'`

Reply:
@mastafit This might help.

Code inputs were intentionally hidden for offline codes: https://github.com/seamapi/react/pull/606

But the current (and updated) logic also hides it for other code kinds. What if we only hide the inputs when for the offline code case?

```
const hasCodeInputs = accessCode.is_offline_access_code !== ture
```

Reply:
Makes sense, I have updated the code!